graveler/committed/apply.go

```diff
@@ -14,6 +14,18 @@ type ApplyOptions struct {
 	AllowEmpty bool
 }
 
+func add(d *graveler.DiffSummary, typ graveler.DiffType, n int) {
+	if d.Count != nil {
+		d.Count[typ] += n
+	}
+}
+
+func inc(d *graveler.DiffSummary, typ graveler.DiffType) {
+	add(d, typ, 1)
+}
```

Comment:
Consider move them into methods of `DiffSummary` or add diffSummary to the function name - package level func named 'add' is a bit general

Reply:
Done.  Using a longer name rather than a method, because these are not really methods and the summary is POD.